test/fixtures/pizza-collaboration/new.bpmn

```diff
@@ -71,7 +71,7 @@
     <semantic:intermediateCatchEvent id="_6-219" name="60 minutes">
       <semantic:incoming>_6-424</semantic:incoming>
       <semantic:outgoing>_6-426</semantic:outgoing>
-      <semantic:timerEventDefinition>
+      <semantic:timerEventDefinition id="_6-220">
```

Comment:
I had to add an id here for the diff not to return undefined. I am not that familiar with the format of bpmn files with the `<semantic/>` notation, but I assume that you are not able to create an element without an id. Please correct me if I am wrong though.

Reply:
I'd rather not force IDs on the diagram as they are not required on the event definitions. Perhaps we need https://github.com/bpmn-io/bpmn-js-differ/issues/20 to be implemented to display the results correctly. For `_changed`, I'd expect then the intermediate catch event ID.